test.rb

```diff
@@ -0,0 +1,18 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+aaa;
```

Comment:
__[rubocop]__
* Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)